game/score.go

```diff
@@ -6,32 +6,14 @@
 package game
 
 type Score struct {
-	RobotsBypassed  [3]bool
-	LeaveStatuses   [3]bool
-	Reef            Reef
-	BargeAlgae      int
-	ProcessorAlgae  int
-	EndgameStatuses [3]EndgameStatus
-	Fouls           []Foul
-	PlayoffDq       bool
+	RobotsBypassed [3]bool
+	LeaveStatuses  [3]bool
+	Mayhem         Mayhem
+	ParkStatuses   [3]bool
```

Comment:
not all games may have "Leave" and "Park" as scoring actions. should those concepts be merged into Mayhem? I'm not sure whether you intend Mayhem to have all the game-specific features